confluent_kafka/src/confluent_kafka.c

```diff
@@ -836,6 +836,33 @@ static void error_cb (rd_kafka_t *rk, int err, const char *reason, void *opaque)
 	CallState_resume(cs);
 }
 
+static int stats_cb(rd_kafka_t *rk, char *json, size_t json_len, void *opaque)
+{
+	Handle *h = opaque;
+	PyObject *eo=NULL, *result=NULL;
```

Comment:
Style is inconsistent, let's try to maintain same style as in existing file to ease readability.

Functions: Open-brace on same line as function decl.
Variable declarations: `type SPACE variablename SPACE = SPACE value;`
If statements: `if SPACE (EXPR) SPACE {`
EXPR: `lhs SPACE operator SPACE rhs` (e.g., `json_len == 0`)